src/ResourceManager/DataLakeStore/Commands.DataLakeStore/help/Add-AzureRmDataLakeStoreFirewallRule.md

```diff
@@ -37,7 +37,7 @@ This creates a new firewall rule called "MyRule" in account "ContosoADL" with an
 The Data Lake Store account to add the firewall rule to
 
 ```yaml
-Type: System.String
```

Comment:
@lewu-msft you'll need to regenerate the help using the [`Update-MarkdownHelpModule` cmdlet with the `-UseFullTypeName` parameter](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module) to ensure that the full type name is preserved in the markdown